Sources/ArcGISToolkit/Components/Popups/PopupView.swift

```diff
@@ -65,7 +65,14 @@ public struct PopupView: View {
                     case .success(_):
                         PopupElementScrollView(popupElements: popup.evaluatedElements)
                     case .failure(let error):
-                        Text("Popup evaluation failed: \(error.localizedDescription)", bundle: .module)
+                        Text(
+                            "Popup evaluation failed: \(error.localizedDescription)",
+                            bundle: .module,
+                            comment: """
+                                     An error message shown when a popup cannot be shown. The
```

Comment:
Just to avoid "shown"-"shown"...
```suggestion
                                     An error message shown when a popup cannot be displayed. The
```

Reply:
@dfeinzimer You OK with this change?